api/envoy/config/filter/http/ext_authz/v2/ext_authz.proto

```diff
@@ -90,6 +90,10 @@ message ExtAuthz {
   //
   // If this field is not specified, the filter will be enabled for all requests.
   api.v2.core.RuntimeFractionalPercent filter_enabled = 9;
+
+  // The labels to provide to the external service in the
+  // :ref:`AttributeContext.Peer.labels<envoy_api_field_service.auth.v2.AttributeContext.Peer.labels>`.
+  repeated string peer_labels = 10;
```

Comment:
I think we have insufficient documentation around labels generally, I just had a read through the referenced link and this PR to try and understand this from first principles. Can you explain where these labels are defined, what values are legitimate, etc.?

Reply:
I've renamed the field to `labels` and added more documentation, including the legitimate values.

Reply:
I have added the docs to the four `ext_authz.proto` files (for http and network filters; v2 and v3alpha versions). That feels correct, because the valid label names could vary per filter-type and API version.  An alternative would have been to document the labels in `attribute_context.proto` but it's not clear to me that this file is specific to external authorization - at least the package name (`envoy.service.auth.v2`) implies that it could be used in other auth cases.